argoverse/data_loading/argoverse_stereo_loader.py

```diff
@@ -0,0 +1,432 @@
+            return self
+
+    def __len__(self) -> int:
+        return len(self.log_list)
+
+    def __str__(self) -> str:
+        frame_image_stereo = self.num_stereo_camera_frame
+
+        num_images = [len(self.image_list[cam]) for cam in STEREO_CAMERA_LIST]
+
+        start_time = self.image_timestamp_list[STEREO_FRONT_LEFT_RECT][0]
+        end_time = self.image_timestamp_list[STEREO_FRONT_LEFT_RECT][-1]
+
+        time_in_sec = (end_time - start_time) * 10 ** (-9)
+        return f"""
+--------------------------------------------------------------------
+------Log id: {self.current_log}
+--------------------------------------------------------------------
+Length of the sequence: {time_in_sec:.2f} seconds
+Number of stereo pair frames (@5 Hz): {frame_image_stereo}
+        """
+
+    def __getitem__(self, key: int) -> "ArgoverseStereoLoader":
+        self.counter = key
+        self.current_log = self.log_list[self.counter]
+        self.num_disparity_frame = len(self.disparity_list)
+        self.num_stereo_camera_frame = len(self.image_timestamp_list[STEREO_FRONT_LEFT_RECT])
+        return self
+
+    def get(self, log_id: str) -> "ArgoverseStereoLoader":
+        """get ArgoverseStereoLoader object with current_log set to specified log_id
+
+        Args:
+            log_id: log id
+        Returns:
+            ArgoverseStereoLoader: with current_log set to log_id
+        """
+        self.current_log = log_id
+        self.num_stereo_camera_frame = len(self.image_timestamp_list[STEREO_FRONT_LEFT_RECT])
+        return self
+
+    def get_image_list(self, camera: str, log_id: Optional[str] = None, load: bool = False) -> List[str]:
+        """Get list of image/or image path
+
+        Args:
+            camera: camera based on camera_stats.STEREO_CAMERA_LIST
+            log_id: log_id, if not specified will use self.current_log
+            load: whether to return image array (True) or image path (False)
+
+        Returns:
+            np.array: list of image path (str or np.array)),
+        """
+        assert self.image_list is not None
+        assert self._image_list is not None
+
+        if log_id is None:
+            log_id = self.current_log
+        if load:
+            return [self.get_image(i, camera) for i in range(len(self._image_list[log_id][camera]))]
+
+        return self._image_list[log_id][camera]
+
+    def get_disparity_list(self, name: str, log_id: Optional[str] = None, load: bool = False) -> List[str]:
+        """Get list of image/or image path
+
+        Args:
+            name: disparity map name, one of:
+
+                ["stereo_front_left_rect_disparity",
+                 "stereo_front_left_rect_objects_disparity"]
+
+            log_id: log_id, if not specified will use self.current_log
+            load: whether to return image array (True) or image path (False)
+
+        Returns:
+            np.array: list of disparity image path (str or np.array)),
```

Comment:
i think this return type `np.array` classes with the `List[str]` return type -- which is preferred?

Reply:
Sure, changed to: 
```python
        Returns:
            List[str]: list of disparity image paths (str),
```